.github/CODEOWNERS

```diff
@@ -261,6 +261,7 @@
 /cmd/systray/                           @DataDog/windows-products
 /cmd/security-agent/                    @DataDog/agent-security
 /cmd/installer/                         @DataDog/fleet @DataDog/windows-products
+/cmd/host-profiler/                     @DataDog/opentelemetry-agent
```

Comment:
Would it make sense to also add @DataDog/profiling-full-host in both, so that both teams are notified / code owners of that logic?

Reply:
Fixed by https://github.com/DataDog/datadog-agent/pull/41001/commits/16be99c1416c104ff431c1b6b66c97ee45352412